src/thor/isochrone.cc

```diff
@@ -217,17 +239,17 @@ void Isochrone::UpdateIsoTile(const EdgeLabel& pred,
     auto tile1 = isotile_->TileId(ll0);
     auto tile2 = isotile_->TileId(ll);
     if (tile1 == tile2) {
-      isotile_->SetIfLessThan(tile1, secs1 * kMinPerSec);
+      isotile_->SetIfLessThan(tile1, {secs1 * kMinPerSec, dist1 * kMetersPerKm});
```

Comment:
the constant looks wrong here, shouldnt it be something like `kKmPerMeter` and be like `0.001f`?